explorer/explorerroutes.go

```diff
@@ -1444,6 +1444,14 @@ func (exp *explorerUI) Search(w http.ResponseWriter, r *http.Request) {
 		http.Redirect(w, r, "/tx/"+searchStr, http.StatusPermanentRedirect)
 		return
 	}
+	if !exp.liteMode {
+		// Search for occurrences of the transaction in the database.
+		dbTxs, _ := exp.explorerSource.Transaction(searchStr)
```

Comment:
Still need to check error.  The only errors we should be OK with are `nil` (when we should redirect) and `sql.ErrNoRows` (which should not throw an internal server error, but also not redirect).  When `err != nil && err != sql.ErrNoRows`, that is an internal server error.  Note that `dbTxs==nil` when `err == sql.ErrNoRows`.